filebeat/module/kafka/log/ingest/pipeline.yml

```diff
@@ -4,8 +4,9 @@ processors:
     field: message
     trace_match: true
     patterns:
-    - (?m)%{TIMESTAMP_ISO8601:kafka.log.timestamp}. %{LOGLEVEL:log.level} +%{JAVALOGMESSAGE:message}
+    - (?m)%{TIMESTAMP_ISO8601:kafka.log.timestamp}. ?%{LOGLEVEL:log.level} +%{JAVALOGMESSAGE:message}
```

Comment:
What is the purpose of the extra `?` before `%{LOGLEVEL:log.level}`?

Reply:
Nothing. This PR was still in progress when you reviewed it. :P :D

Reply:
Sorry, @andresrc asked for some eyes on it so I reviewed it.